projects/example-app/src/app/books/actions/book-exists-guard.actions.ts

```diff
@@ -0,0 +1,14 @@
+import { Action } from '@ngrx/store';
+import { Book } from '../models/book';
+
+export enum BookExistsGuardActionTypes {
```

Comment:
This should remain `book.actions.ts` with the context of where the action came from as `[Book Exists Guard]`